lineage.go

```diff
@@ -268,3 +279,23 @@ type unaryConvLineage[T Assignee] struct {
 func (lin *unaryConvLineage[T]) TypedSchema() TypedSchema[T] {
 	return lin.tsch
 }
+
```

Comment:
Need comment for exported func. Probably worth noting in that comment that the comparison algorithm this currently uses is unsophisticated and subject to change